client_test/directfund_test.go

```diff
@@ -81,12 +78,12 @@ func TestWhenObjectiveIsRejected(t *testing.T) {
 
 	response := clientA.CreateDirectChannel(request)
 
-	waitTimeForCompletedObjectiveIds(t, &clientB, time.Second, response.Id)
+	waitTimeForCompletedObjectiveIds(t, &clientA, time.Second, response.Id)
 
 	obj, _ := storeA.GetObjectiveById(response.Id)
 
-	if obj.GetStatus() != protocols.Approved {
```

Comment:
It looks like this test was asserting incorrect behaviour, and passing because the client was exhibiting the same incorrect behaviour? Is that right?

I suppose in an ideal world the test would have asserted correct behaviour, and been skipped. And then you could have just unskipped it here?

Reply:
Skipping the intended test seems totally reasonable to me. I think that the old behaviour was somewhat intentional though?